tekton/rpmbuild/tekton.spec

```diff
@@ -0,0 +1,619 @@
+%define debug_package %{nil}
+%define repo github.com/tektoncd/cli
+%define version _VERSION_
+
+%if %{undefined gobuild}
+%define gobuild(o:) go build -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n')" -a -v -x %{?**};
+%endif
+
+Name:           tkn
+Version:        %{version}
+Release:        1%{?dist}
+Summary:        A CLI for interacting with Tekton!
+License:        Apache 2.0
+URL:            https://%{repo}
+
+BuildRequires: golang >= 1.10
+BuildRequires: git
+
+%global import_path github.com/tektoncd/cli
+
+Source0:        https://%{repo}/archive/v%{version}.tar.gz
+
+%description
+The Tekton Pipelines cli project provides a CLI for interacting with Tekton !
+
+# vendored libraries (Source0)
```

Comment:
Are those generated with a tool ?

Reply:
@khrm   was saying so

what about we merge (when i finish to iterate on it) this and let khuram updating it with automatic generation of the deps if needed,

Reply:
@chmouel yeah works for me :wink: